lib/OpenQA/Schema/Result/GruDependencies.pm

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2015 SUSE Linux Products GmbH
```

Comment:
Sorry to bother you again, but this company name is no longer valid.
For 2015 `Copyright (C) 2015 SUSE Linux GmbH` or `Copyright (C) 2015 SUSE LLC` should be used. Preferably the latter.